src/main/java/nextstep/subway/global/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.subway.global;
+
+import java.util.NoSuchElementException;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
+
+    @ExceptionHandler(DataIntegrityViolationException.class)
```

Comment:
동일한 응답을 반환하는 예외는 다음과 같이 처리할 수도 있습니다.
```suggestion
    @ExceptionHandler({DataIntegrityViolationException.class, IllegalArgumentException.class})
```